### ISSUE TITLE
download.sh

### ISSUE THREAD

**Issue by tommiesatellite** [google-coral/edgetpu#177]
running download_imprinting_test_data.sh, the script cannot find download.sh

**Comment by Namburger** [comment]
Hey! 
download.sh is here: https://github.com/google-coral/edgetpu/blob/master/scripts/download.sh
You need to be in the examples directory of this repo in order for the script to work.

**Comment by Namburger** [comment]
@tommiesatellite 
FYI, the `open_image_v4_subset.csv` file have a line that has an expired link, this may also cause a download problem. 
Please remove [this line](https://farm8.staticflickr.com/4046/4413074160_a28a427470_o.jpg,Parrot,a88a4d64507a6501ee09606ce8e57545b70d1cd673f766b46e879dd97e121a55) from that file!

**Comment by Namburger** [comment]
@tommiesatellite actually, it's probably better that you just use the download instructions [here](https://coral.ai/docs/edgetpu/retrain-classification-ondevice/#retrain-a-model-on-device-with-our-sample):
```
wget https://dl.google.com/coral/sample_data/imprinting_data_script.tar.gz
tar zxf imprinting_data_script.tar.gz
./imprinting_data_script/download_imprinting_test_data.sh ./
```

**Comment by tommiesatellite** [comment]
Thank you Nam,
Yes, those instructions are definetly the way to go.  I got it running yesterday going through the steps on the imprinting_learning.py program on github.  My problem was mainly that I needed to use sudo before a few an a few of those.  I has been awhile since I worked in Linux so am a bit rough on obvious stuff.  Your tip on the link to one of the images helped.  I got all the rest of the images.
regards,
Tom
